ansible/roles/ooni-backend/tasks/populate_bouncer_base.yml

```diff
@@ -15,13 +15,32 @@
     - "{{ bind_collector_hsdir }}"
     - "{{ bind_web_connectivity_hsdir }}"
 
-- name: Template "{{ bind_bouncer_file  }}"
+- name: Template "{{ bind_bouncer_file }}" with bouncer_base.yaml.j2
   template: >
     src="bouncer_base.yaml.j2"
     dest="{{ bind_bouncer_file }}"
     backup="yes"
+    unsafe_writes="true"
+  when: ( oonibackend_bundle is defined and oonibackend_bundle == True )
+  register: bouncer_config_file
 
-- name: Fetch backend related config files and HS keys
+- name: Template "{{ bind_bouncer_file }}" with "{{ bouncer_base_template }}"
+  template: >
+    src="{{ bouncer_base_template }}"
+    dest="{{ bind_bouncer_file }}"
+    backup="yes"
+    unsafe_writes="true"
+  when: ( oonibackend_bundle is undefined or oonibackend_bundle == False )
```

Comment:
Sure, I guess my point was that there are usages of the default filter. Just use in this one the better syntax of casting to bool that is more robust and compact and you can address the other ones in another pull request.